Source/Train-Railway/Clock.cs

```diff
@@ -8,5 +8,42 @@ namespace Train_Railway
 {
```

Comment:
Interesting that your group added a custom clock instead of using DateTime hh:mm.